### PR TITLE
fix simulator unfocused

### PR DESCRIPTION
**Pull request by PPpro**
resolve:  https://github.com/cocos-creator/3d-tasks/issues/5437

referenced: https://developer.apple.com/documentation/appkit/nsapplication/activationoptions/1525559-activateignoringotherapps

### CODE REVIEW

**Comment by PPpro** [comment]
@minggo  please  review

**Review by minggo — APPROVED** [review]

File: tools/simulator/frameworks/runtime-src/proj.ios_mac/mac/SimulatorApp.mm [hunk 1]
```diff
@@ -383,7 +383,9 @@ - (void) createWindowAndView
     _window.contentView = viewController.view;
     [_window.contentView setWantsBestResolutionOpenGLSurface:YES];
     [_window makeKeyAndOrderFront:nil];
-
+    
+    NSApplication *thisApp = [NSApplication sharedApplication];
+    [thisApp activateIgnoringOtherApps:YES];
```
Comment (minggo) [inline tools/simulator/frameworks/runtime-src/proj.ios_mac/mac/SimulatorApp.mm:388]: Why need it? What does it do?

Comment (PPpro) [reply tools/simulator/frameworks/runtime-src/proj.ios_mac/mac/SimulatorApp.mm:388]: the App Windows on MacOS wouldn't  show  up on the front without  calling  this  API

Comment (PPpro) [reply tools/simulator/frameworks/runtime-src/proj.ios_mac/mac/SimulatorApp.mm:388]: see this issue: https://github.com/cocos-creator/3d-tasks/issues/5437

**Pull request merged**